kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/KieServerRouter.java

```diff
@@ -111,11 +112,13 @@ public void start(String host, Integer port) {
         pathHandler.addExactPath("/containers", new ContainersHttpHandler(notFoundHandler, adminHandler));
         pathHandler.addPrefixPath("/admin", adminHandler);
         pathHandler.addExactPath("/", new KieServerInfoHandler());
-   
+
+        HttpHandler blockingHandler = new BlockingHandler(pathHandler);
+
         // main server configuration
         server = Undertow.builder()
                 .addHttpListener(port, host)                
-                .setHandler(pathHandler)
+                .setHandler(blockingHandler)
```

Comment:
Does this change need a test added? In KieServerRouterBaseIntegrationTest or something?